### PR TITLE
1015 migration chapter updates

### PR DESCRIPTION
**Pull request by sharath-jk**

### CODE REVIEW

**Comment by rssag** [comment]
losing track - how does this relate to the PR for 10.9.0 here? https://github.com/SoftwareAG/c8y-docs/pull/632

File: content/edge/migration-bundle/sample-scripts-to-automate-the-migration.md [hunk 1]
```diff
@@ -9,53 +9,53 @@ layout: redirect
...
+5. Move the ZIP archive with the migration data to your Edge 10.9 appliance.
 
 #### In 10.9 appliance
 
 {{< c8y-admon-important >}}
 Before copying the backup, ensure that there is sufficient disk space in your Edge 10.9 appliance. For example, in the Edge 10.9 appliance, if the size of the data disk is 75 GB and the size of the MongoDB backup is 100 GB, you must expand the size of the data disk to additional 100 GB before copying the MongoDB backup. For more information about disk size expansion, see [Expanding the disk size](/edge/configuration/#expanding-the-disk-size).
 {{< /c8y-admon-important >}}
 
-1. Log in as **root** user.
+1. Log in as root user.
```
Comment (rssag) [inline content/edge/migration-bundle/sample-scripts-to-automate-the-migration.md:58]:
```suggestion
1. Log in as a root user.
```

File: content/edge/migration-bundle/creating-backup-on-107.md [hunk 2]
```diff
@@ -4,29 +4,28 @@ title: Creating a backup on Edge 10.7
 layout: redirect
 ---
 
-In your Edge 10.7 setup, you must back up the data for each tenant and docker collection, and note down the device ID.
+In your Edge 10.7 setup, back up the data for each tenant and docker collection, and note down the device ID.
 
-{{< c8y-admon-important >}}
-Before you back up the data, ensure that there is sufficient disk space to save the backup in your Edge 10.7 appliance. The MongoDB backup requires the same amount of space as the database. For example, if the size of the database is 100 GB, the MongoDB backup also requires 100 GB of disk space. You would need additional 100 GB of disk space to save the MongoDB backup in your Edge 10.7 appliance.  
+{{< c8y-admon-important >}} Before the data back up, ensure that there is sufficient disk space to save the back up in your Edge 10.7 appliance. The MongoDB backup requires the same amount of space as the database. For example, if the size of the database is 100 GB, the MongoDB backup also requires 100 GB of disk space. You would need additional 100 GB of disk space to save the MongoDB backup in your Edge 10.7 appliance.
```
Comment (rssag) [inline content/edge/migration-bundle/creating-backup-on-107.md:9]:
```suggestion
{{< c8y-admon-important >}} Before the data back up, ensure that there is sufficient disk space to save the back up in your Edge 10.7 appliance. The MongoDB backup requires the same amount of space as the database. For example, if the size of the database is 100 GB, the MongoDB backup also requires 100 GB of disk space. You would need an additional 100 GB of disk space to save the MongoDB backup in your Edge 10.7 appliance.  
```

**Review by rssag — APPROVED** [review]

**Comment by sharath-jk** [comment]
@rssag - the migration is supported only for 10.7 to 10.9. The procedure is same for Edge version 10.9, 10.11, 10.13 and 10.15. If a customer on 10.7 has to upgrade to higher version, they must migrate to 10.9 first and then upgrade to a higher version. 

I have created 2 PRs because we did not have important shortcodes in 10.9 version.

**Comment by rssag** [comment]
> @rssag - the migration is supported only for 10.7 to 10.9. The procedure is same for Edge version 10.9, 10.11, 10.13 and 10.15. If a customer on 10.7 has to upgrade to higher version, they must migrate to 10.9 first and then upgrade to a higher version.
> 
> I have created 2 PRs because we did not have important shortcodes in 10.9 version.

should it be cherry-picked to 10.10.0 up until 10.15.0 then?
there's probably some shortcodes missing (admonition boxes, in any case) in some other versions as well.

**Comment by sharath-jk** [comment]
> > @rssag - the migration is supported only for 10.7 to 10.9. The procedure is same for Edge version 10.9, 10.11, 10.13 and 10.15. If a customer on 10.7 has to upgrade to higher version, they must migrate to 10.9 first and then upgrade to a higher version.
> > I have created 2 PRs because we did not have important shortcodes in 10.9 version.
> 
> should it be cherry-picked to 10.10.0 up until 10.15.0 then? there's probably some shortcodes missing (admonition boxes, in any case) in some other versions as well.

Not until 10.15. Only to 10.10.0. 

The changes that we merge to develop as part of this PR can be cherry-picked to 10.15, 10.14, 10.13 and 10.11 versions.

**Pull request merged**